cwms_radar_api/src/main/java/cwms/radar/api/LevelsController.java

```diff
@@ -39,39 +32,34 @@
 import cwms.radar.formatters.xml.adapters.ZonedDateTimeAdapter;
 import io.javalin.apibuilder.CrudHandler;
 import io.javalin.http.Context;
-import io.javalin.plugin.openapi.annotations.*;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiParam;
+import io.javalin.plugin.openapi.annotations.OpenApiRequestBody;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
 import org.jetbrains.annotations.NotNull;
 import org.jooq.DSLContext;
 
+import static com.codahale.metrics.MetricRegistry.name;
+import static cwms.radar.data.dao.JooqDao.getDslContext;
+
 public class LevelsController implements CrudHandler {
     private static final Logger logger = Logger.getLogger(LevelsController.class.getName());
     private final MetricRegistry metrics;
-    private final Meter getAllRequests;
-    private final Timer getAllRequestsTime;
-    private final Meter getOneRequest;
-    private final Timer getOneRequestTime;
+
     private final Histogram requestResultSize;
-    private final Meter createRequest;
-    private final Timer createRequestTime;
-    private final Meter deleteRequest;
-    private final Timer deleteRequestTime;
-    private final Meter updateRequest;
-    private final Timer updateRequestTime;
+
 
     public LevelsController(MetricRegistry metrics){
         this.metrics=metrics;
-        String className = this.getClass().getName();
```

Comment:
are you intentionally removing these, and if so to what purpose? (or am I blind and just not seeing that we've discovered a cleaner way?)

Reply:
The MetricRegistry already holds references to previously created meters, timers etc so we don't have to keep our own references to them.  The method markAndTime looks up in the registry the meter and timer for the provided subject and marks the meter and starts the timer.  I did the same to the RatingController.  I think it makes things cleaner and would like to transition away from every CrudHandler having to have the meter/timer fields.

Reply:
oh, excellent.